collection-scripts/gather_serving

```diff
@@ -10,6 +10,8 @@ resources+=("knativeservings" "configurations" "routes.serving.knative.dev" "ser
 resources+=("authconfigs" "authorinos")
 # Dependent resources NIM
 resources+=("accounts.nim.opendatahub.io")
+# Dependent resources llm-d
+resources+=("llminferenceserviceconfigs" "llminferenceservices" "istiocnis" "leaderworkersets" "istiocni" "gateways")
```

Comment:
```suggestion
resources+=("llminferenceserviceconfigs" "llminferenceservices" "leaderworkersets"  "gateways" "gatewayclasses" "httproutes" "inferencepools")
```

Reply:
You might also want to add `destinationrules` to OSSM section

Reply:
maybe nit, but I'd expand the gateway to be the full name to not have Istio gateway resources back

```
$ k api-resources | grep gateways
gateways                              gtw                                                                                    gateway.networking.k8s.io/v1                       true         Gateway
gateways                              gw                                                                                     networking.istio.io/v1beta1                        true         Gateway
```

Reply:
fixed